crate/server/src/config/command_line/db.rs

```diff
@@ -177,8 +177,9 @@ impl DBConfig {
                         "KMS_REDIS_MASTER_PASSWORD",
                     )?;
                     // Generate the symmetric key from the master password
-                    let master_key =
-                        RedisWithFindex::master_key_from_password(&redis_master_password)?;
+                    // TODO - store salt somewhere ?
```

Comment:
IMHO, KMS server should not handle salts for customers unless this feature is specifically required. Customers should be informed as soon as possible (at the top level of salt use, e.g in the CLI) that provided salts are (or are not) FIPS compliant